### ISSUE TITLE
[BUG] ColorHelper.getThemeColorHexString("primary") returns wrong color.

### ISSUE THREAD

**Issue by JacobOJ** [kirbydesign/designsystem#3159]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
ColorHelper.getThemeColorHexString("primary") returns the same color no matter which bank is being used.
I would expect it to return the brand specific primary color for the current context.

## Describe how to reproduce the bug
1. call ColorHelper.getThemeColorHexString("primary").
2. It returns #00e89a
3. It should return brand specific colors for the current bank.

## Which Kirby version was used?
8.8.0

## What was the expected behavior?
I expect brand specific colors to be returned.

## Please complete the following information:
- OS: MacOS
- Browser Firefox
- Browser version 114

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
